bundles/AdminBundle/Resources/public/js/pimcore/asset/video.js

```diff
@@ -111,120 +117,147 @@ pimcore.asset.video = Class.create(pimcore.asset.asset, {
 
             var date = new Date();
 
-            this.previewImagePanel = new Ext.Panel({
-                width: 300,
-                region: "east",
-                scrollable: 'y',
+            var detailsData = [];
+            detailsData[t("width")] = this.data.customSettings.videoWidth;
+            detailsData[t("height")] = this.data.customSettings.videoHeight;
+            detailsData[t("duration")] = this.data.customSettings.duration;
+
+            var details = [];
+
+            details.push({
```

Comment:
It's not necessary to break this up completely, just create the `dimensionPanel` and add it to the desired place, e.g.: 

```javascript
items: [dimensionPanel, {
...
...
```